src/UbiProtocolHandler.cpp

```diff
@@ -95,15 +94,19 @@ void UbiProtocolHandler::add(const char *variable_label, float value, char *cont
 bool UbiProtocolHandler::send(const char *device_label, const char *device_name) {
   // Builds the payload
   char *payload = (char *)malloc(sizeof(char) * MAX_BUFFER_SIZE);
-  if (_iotProtocol == UBI_TCP || _iotProtocol == UBI_UDP) {
+  if (_iotProtocol == UBI_TCP) {
     buildTcpPayload(payload, device_label, device_name);
   } else {
-    buildHttpPayload(payload);
+    if (_debug) {
+      Serial.println(F("[ERROR] Wrong IoT Protocol. Only UBI_TCP supported"));
```

Comment:
There is a typo in the message, please change it to this:

[ERROR] wrong IoT Protocol. Only UBI_TCP is supported by this library.